lib/algos/list/skip.js

```diff
@@ -10,10 +10,10 @@ const MAX_STREAK_LENGTH = 100;
  */
 class Skip {
     /**
-     * @param {Object} params           - skip parameters
-     * @param {Object} params.extension - delimiter extension used (required)
-     * @param {String} params.gte       - current range gte (greater than or
-     *                                    equal) used by the client code
+     * @param {Object} params                - skip parameters
+     * @param {Object} params.extension      - delimiter extension used (required)
+     * @param {String | string[]} params.gte - current range gte (greater than or
```

Comment:
This change is related to the fact that in MongoClientInterface we actually pass an array here: the function type was not properly defined here, leading to TS error. Having an array here is fine and expected.